### DISCUSSION TITLE
Could we please have an updated Linux repository for osu!lazer?

### DISCUSSION THREAD

**Discussion by TibixDev** [ppy/osu#13916] [Ideas] [answered]
I currently rely on using the AUR version of osu!lazer that is almost never up to date, and multiplayer can only be played with the latest release.

Since appimages are not a proper way to run things on Linux could you please create some sort of updated repository for osu!lazer? I, and the rest of the Linux players would probably really appreciate it.

Have a nice day. :)

**Comment by gagahpangeran** [comment]
> Since appimages are not a proper way to run things on Linux

Why is this not a proper way to run things on Linux?

The idea of using AppImages because it is distro agnostic and you can just download and run the file.

We don't have resources to maintain update to every single repository for every single distro.

**Comment by peppy** [comment, marked as answer]
We have a grace period for older releases where multiplayer is still possible. You must be doing something horribly wrong.

Apart from this, what has already been said by @gagahpangeran stands. AppImage is our distribution format of choice for the time being. If you can't handle this, please use a different operating system.

**Reply by bdach** [reply]
The user mentioned using AUR packages. I don't even know what's going with them. It's a mess.

There's the old https://aur.archlinux.org/packages/osu-lazer-git/, not updated for ages, and which was built from source (therefore not supporting multiplayer), and then there are https://aur.archlinux.org/packages/osu-lazer/ and https://aur.archlinux.org/packages/osu-lazer-bin/ which I don't even know *what* they are or why there's two of them.

I don't think we maintain any of these, and therefore there's nothing that can be done with those on our side.

**Reply by smoogipoo** [reply]
`osu-lazer-git` and `osu-lazer` compile locally on the target machine, as is typically the case with `-git` packages.

`osu-lazer-bin` uses the AppImage, which is the correct version to use for multiplayer.
